components/eam/src/physics/crm/cospsimulator_intr.F90

```diff
@@ -1590,6 +1590,13 @@ subroutine populate_cosp_subcol(npoints, state, pbuf, emis, cld_swtau, cospstate
            end do
          end do
        end do
+       if (present(snow_tau) .and. present(snow_emis)) then
+          dem_s_snow(1:npoints,1:pver)  = snow_emis(1:npoints,1:pver)  ! 10.5 micron grid-box mean optical depth of stratiform snow
```

Comment:
I think this might actually cause more problems; snow_emis and snow_tau have size (pcols,pver), but npoints can actually be larger than pcols for MMF (ncol*crm_nx_rad*crm_ny_rad). I think maybe we should change the declaration of snow_emis and snow_tau to size npoints to allow this.